package.json

```diff
@@ -19,6 +19,8 @@
     "docs": "cd docs; make html;",
     "lint": "jshint *.js packages",
     "test": "mocha; jshint *.js packages",
+    "e2e": "docker-compose down && docker-compose build && docker-compose run --entrypoint 'npm run e2e:mocha' test_runner",
+    "e2e:mocha": "mocha --recursive ./e2e/**/*.spec.js -R spec --exit",
```

Comment:
I'm not sure of introducing a new testing framework. What's the reason? 
I was struggling with the Jest magic in the project but adding yet an other framework might increase package complexity even more.